File: Java/Driver.java
package Java;

public class Driver extends Account {
    public Driver(String name, String document, String email, String password){
        super(name, document, email, password);
    }

    void printDataDriver(){
        System.out.println("Document User: " + document + " Name: " + name + " e-Mail: " + email + " Password: " + password);
    }
}

File: JS/Driver.js
class Driver extends Account{
    constructor(name, document, email, password){
        super(name, document, email, password)
    }
    printDataUser(){
        console.log("Nombre: " + this.name)
        console.log("Documento: " + this.document)
        console.log("e-Mail: " + this.email)
        console.log("Password: " + this.password)
    }
}